Dockerfile

```diff
@@ -42,6 +42,8 @@ ARG PKG_CNI
 ARG PKG_FLANNEL_CNI
 ARG PKG_TALOSCTL_CNI_BUNDLE_INSTALL
 
+ARG DEBUG_TOOLS_SOURCE
```

Comment:
this is not used anywhere below, so all the debug stuff would get copied always which we don't want